providers/implementations/serializers/serializer_common.c

```diff
@@ -200,40 +200,45 @@ int ossl_prov_print_labeled_bignum(BIO *out, const char *label,
      * first limb.
      * i is used as limb index, j is used as the "byte index" in the limb
      */
-    for (i = bytes / BN_BYTES - 1; i >= 0; i--) {
+    for (i = (bytes + (BN_BYTES - 1)) / BN_BYTES - 1; i >= 0; i--) {
```

Comment:
@levitte - I had to modify this as it only worked properly for BN_BYTE aligned data.. The data I was testing was 28 bytes which meant that the top long was discarded. The z variable was taken from BN_bn2hex() in order to skip the top unaligned zeros.

Reply:
Sigh: The windows appveyor problem has convinced me that this really was the wrong approach. I have changed it to use BN_bintohex which should - 
(1) Actually work
(2) Be simpler.

I will see if that resolves the appveyor issue.